### DISCUSSION TITLE
Github Support Not Responding to ticket for Non-Profit account

### DISCUSSION THREAD

**Discussion by bgibson-cdchoices** [community/community#122744] [Enterprise]
We submitted a ticket via the link to request the non-profit benefit on our account on the [discount plans page](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/discounted-plans-for-github-accounts) a few weeks ago and other than the email acknowledging that it has been received there has been no indication that it's even been received by the correct team. Even though it says we would be refunded the last transaction if we pay for a month, I'm hesitant to do so if it takes longer than a single billing cycle to get approved. 

Is there an alternative method other than tickets to contact GitHub support? Is this type of wait time typical? We had been evaluating getting an enterprise plan but wanted to establish our use with a team account before making the final decision whether we were going to move forward with enterprise or not.

**Comment by baharbaba** [comment]
I've been waiting for a response for 2 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.
